docs/src/xhtml/components/asides/index.xhtml

```diff
@@ -75,6 +75,27 @@
 								</figure>
 							</section>
 
+							<h3>Adding a footer</h3>
+							<section>
+								<p>
+									You can insert a footer in the Aside. Note that
+									<a data-ts="Button" onclick="toggle('#myfooter');">the footer</a> will fixed at
```

Comment:
This should be `will be fixed` I guess :thinking:

Reply:
👍